README.md

```diff
@@ -60,3 +60,30 @@ Full instructions can be found [here](test/README.md#local-usage).
 ### Testing
 
 For details on how to conduct tests on StorageHub, please see the testing [README.md](test/README.md).
+
+## FAQ
+
+### I've just updated a RuntimeApi or RPC call, how do I update the fn signatures in polkadot{.js} API/App?
+
+Navigate to the `/types-bundle` package and make your changes to `/src/rpc.ts` and `/src/runtime.ts`.
+
+Any new Structs or ErrorEnums can be defined at `/src/types.ts` , using existing examples for guidance, and any new branded types as well.
+
+Afterwards in the directory, run:
+
+```sh
+pnpm i 
+pnpm build
+pnpm fmt:fix
+```
```

Comment:
Can we bundle this into `pnpm typegen` so that it is quicker/easier to remember for devs?

Reply:
Done